logstash-scheduler/src/main/java/org/apache/mesos/logstash/config/LogstashConfig.java

```diff
@@ -9,9 +9,16 @@
 @Component
 @ConfigurationProperties(prefix = "logstash")
 public class LogstashConfig {
+
+    private static final String DEFAULT_EXECUTOR_IMAGE_NAME = "mesos/logstash-executor";
```

Comment:
I thought about this, but decided to keep them in order to ease implementation of methods like `isDefaultContainer()`... well, I would keep them if you don't mind. No problems with removing them either. Let me know, plz